Restructure docs (#374)
kyma-project/application-connector-manager: docs/user/tutorials/01-00-integrate-external-system.md

# Integrate an External System with Kyma

## Introduction

> The document describes the steps needed to connect an external system (for example, HTTPBin) with a Kyma runtime.

Reviewer: ```suggestion
The document describes the steps needed to connect an external system (for example, HTTPBin) with Kyma runtime.
```

In this example, Kyma sends authenticated requests to an external service.

## Prerequisites

1. Create Kyma runtime either by using the [SAP BTP cockpit](https://help.sap.com/docs/btp/sap-business-technology-platform/create-kyma-environment-instance) or by following the [Kyma Quick Install](https://kyma-project.io/#/02-get-started/01-quick-install) tutorial.
2. Besides the Kyma default modules like Istio and API Gateway, you must enable the following Kyma modules:

>     * `Application Connector`: it includes the Application Gateway for proxing requests to external systems.

Reviewer: ```suggestion
    * Application Connector: it includes the Application Gateway for proxying requests to external systems.
```

# Integrate an External System with Kyma

## Introduction

The document describes the steps needed to connect an external system (for example, HTTPBin) with a Kyma runtime.

In this example, Kyma sends authenticated requests to an external service.

## Prerequisites

> Besides the Kyma default modules like Istio and API Gateway, you must enable the following Kyma modules:

Reviewer: ```suggestion
```

1. Create Kyma runtime either by using the [SAP BTP cockpit](https://help.sap.com/docs/btp/sap-business-technology-platform/create-kyma-environment-instance) or by following the [Kyma Quick Install](https://kyma-project.io/#/02-get-started/01-quick-install) tutorial.

> 2. Add following Kyma modules:

Reviewer: ```suggestion
2. Besides the Kyma default modules like Istio and API Gateway, you must enable the following Kyma modules:
```

    * `Application Connector`: it includes the Application Gateway for proxing requests to external systems.

>     * `Serverless`: used to run a Function which is sending an HTTP request to the external system.

Reviewer: ```suggestion
    * `Serverless`: used to run a Function which is sending an HTTP request to an external system.
```

>     If you created the Kyma runtime via BTP, follow this tutorial for [Adding and Deleting a Kyma Module](https://help.sap.com/docs/btp/sap-business-technology-platform/enable-and-disable-kyma-module?#add-and-delete-a-kyma-module-using-kyma-dashboard). Alterantively, continue with the steps described in the [Kyma Quick Install](https://kyma-project.io/#/02-get-started/01-quick-install?id=steps) tutorial.

Reviewer: ```suggestion
    If you created Kyma runtime using SAP BTP cockpit, follow this tutorial for [Adding and Deleting a Kyma Module](https://help.sap.com/docs/btp/sap-business-technology-platform/enable-and-disable-kyma-module?#add-and-delete-a-kyma-module-using-kyma-dashboard). Alternatively, continue with the steps described in the [Kyma Quick Install](https://kyma-project.io/#/02-get-started/01-quick-install?id=steps) tutorial.
```
